C/services/south/south_plugin.cpp

```diff
@@ -68,7 +68,7 @@ Reading SouthPlugin::poll()
 /**
  * Call the reconfigure method in the plugin
  */
-void SouthPlugin::reconfigure(string& newConfig)
+void SouthPlugin::reconfigure(const string& newConfig)
 {
 	return this->pluginReconfigurePtr(instance, newConfig);
```

Comment:
The south plugins don't have 2nd param as const in:

plugin_reconfigure(PLUGIN_HANDLE *handle, string& newConfig)

But I guess it is not possible to check const-correctness (or even param type) in this case anyway, since we are using dlopen/dlsym mechanism to call plugin_* functions.